racket/src/racket/src/struct.c

```diff
@@ -5999,8 +6024,9 @@ static Scheme_Object *do_chaperone_struct(const char *name, int is_impersonator,
       ("(or/c " extra "struct-accessor-procedure?\n"      \
        "      struct-mutator-procedure?\n"                 \
        "      struct-type-property-accessor-procedure?\n"  \
+       "      struct-type-property-predicate-procedure?\n"  \
```

Comment:
Is this intended? I don't think property predicates are allowed as arguments to `chaperone-struct` or `impersonate-struct`, but I could be lost here.